src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/ui_get_capabilities_response_test.cc

```diff
@@ -277,7 +277,7 @@ TEST_F(UIGetCapabilitiesResponseTest, SetVideoStreamingCapability_SUCCESS) {
   video_streaming_capability[strings::preferred_resolution]
                             [strings::resolution_width] = 800;
   video_streaming_capability[strings::preferred_resolution]
-                            [strings::resolution_height] = 350;
+                            [strings::resolution_height] = 354;
```

Comment:
@IGapchuk Why did you change this value from 350->354 ? Does it make sense ?

Reply:
@ValeriiMalkov That changes has been did according to proposal example:
Example 1: 8” screen (800x480 pixels)

scale (1x)
viewport of the head unit (800x354 pixels)
viewport of the apps view (800x354 points)
pixel per inch (117 ppi)
points per inch (117 ppi)